src/api/TaskLinks.js

```diff
@@ -5,9 +5,30 @@
  */
 
 import Base from './Base';
-import { ERROR_CODE_CREATE_TASK_LINK } from '../constants';
+import {
+    ERROR_CODE_CREATE_TASK_LINK,
+    HTTP_STATUS_CODE_RATE_LIMIT,
+    HTTP_STATUS_CODE_INTERNAL_SERVER_ERROR,
+    HTTP_STATUS_CODE_NOT_IMPLEMENTED,
+    HTTP_STATUS_CODE_BAD_GATEWAY,
+    HTTP_STATUS_CODE_SERVICE_UNAVAILABLE,
+    HTTP_STATUS_CODE_GATEWAY_TIMEOUT,
+} from '../constants';
+
+const RETRYABLE = [
```

Comment:
Is there any shared constants area for tasks? Could be useful since these classes will probably keep retryable fields in sync.